airflow/providers/edge/openapi/edge_worker_api_v1.yaml

```diff
@@ -0,0 +1,102 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+openapi: 3.0.2
+info:
+  title: Airflow Edge Worker API
+  version: 0.1.0
```

Comment:
A question to the audience -
Is there a current policy for version management of APIs in providers? Should we couple it with the provider's version?

Reply:
I believe we never made a formal policy for this. I assume this is per provider/API. But there are not many different, besides internal API there was just one added for FAB.

Anyway with Airflow 3 this will need to be re-worked to FastAPI anyway, this API is pretty "machine driven" from the RPC code, like internal API. I would also treat this as "internal" for the consumption inside the provider, not to be expected to be exposed/stable. RPC calls will be the main point of variation. Aligned it with internal API and updated the version to be 1.0.0.